store/transfer_processor.go

```diff
@@ -169,17 +167,19 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 			to := gethCommon.BytesToAddress(e.To)
 			contractsAddrs[contractAddr][to] = struct{}{}
 		}
-	}
 
-	// Make sure coinbases are also included in balance queries to geth
-	for _, addr := range coinbases {
-		if allSubs[addr] == nil {
-			continue
-		}
-		if contractsAddrs[model.ETHAddress] == nil {
-			contractsAddrs[model.ETHAddress] = make(map[gethCommon.Address]struct{})
+		if e.IsMinerRewardEvent() {
+			// only one event on miner reward
+			totalMinerReward.SetString(e.Value, 10)
```

Comment:
double check we only see miner and uncle reward event one time each per block

Reply:
ok